src/Npgsql.NodaTime/Internal/TimestampConverters.cs

```diff
@@ -49,7 +49,7 @@ protected override ZonedDateTime ReadCore(PgReader reader)
 
     protected override void WriteCore(PgWriter writer, ZonedDateTime value)
     {
-        if (value.Zone != DateTimeZone.Utc && !LegacyTimestampBehavior)
+        if (value.Zone != DateTimeZone.Utc)
```

Comment:
Why remove? Shouldn't it be possible to write non-UTC ZonedDateTime in legacy mode?

Reply:
Right, I didn't see they were shared.